tests/unit/testTypesService.unitTest.ts

```diff
@@ -324,6 +324,29 @@ describe("when database is on", () => {
       }
     );
   });
+
+  context("when the suggested test type ids are present", () => {
+    it("should return the suggested test type ids", () => {
+      return testTypesService
+        .getTestTypesById("1", {
+          fields: [
+            "testTypeClassification",
+            "defaultTestCode",
+            "linkedTestCode",
+          ],
+          vehicleType: "psv",
+          vehicleConfiguration: "articulated",
+        })
+        .then((returnedRecords: any) => {
+          expect(returnedRecords).toEqual({
+            id: "1",
+            testTypeClassification: "Annual With Certificate",
+            defaultTestCode: "adl",
+            linkedTestCode: null,
+          });
+        });
+    });
+  });
```

Comment:
This test needs to be removed as it is not valid anymore.

Reply:
I had a look at the FE story: https://jira.dvsacloud.uk/browse/CVSB-11706
so it seems we are not removing id from the the payload so i believe this test is still valid as the list get ordered by id once first loaded then the modal window will use the new field to suggest the relevant test-types when a test type is added.
Please see this: https://github.com/dvsa/cvs-app-mobile/blob/f4fa0bc82951d3ee183cb6a85f34a05b91edd8e0/src/pages/testing/test-creation/test-types-list/test-types-list.ts#L49

Reply:
@JonLazariniDD this test was added for `getTestTypesById` function  for `suggestedTestTypeId` but this function doesn't need to return `suggestedTestTypeId` so the test isn't needed.

Reply:
Thanks for clarifying and spotting it.
We added it an additional test for `getTestTypesList` which is now checking the payload since the api schemas are contracts between this service and FE applications.